src/targets/gpu/fuse_ops.cpp

```diff
@@ -310,6 +331,115 @@ void move_standard_front(std::vector<instruction_ref>& args)
         std::swap(*it, args.front());
 }
 
+struct find_gelu
+{
+
+    static auto erf_fn()
+    {
+        return match::name("gpu::erf")(
+            match::arg(0)(match::used_once(),
+                          match::name("gpu::mul")(match::arg(0)(match::any().bind("x")),
+                                                  match::arg(1)(match::has_value(M_SQRT1_2)))));
```

Comment:
Multiplication is commutative, so you should use `match::either_arg(0, 1)`. Instead of `match::any()`, you might need to do `match::none_of(match::has_value(M_SQRT1_2))`.